tests/test_diffeq/test_ode/test_ivp_examples.py

```diff
@@ -0,0 +1,156 @@
+        rv = Constant(0.1)
+        lg1 = ivp_examples.logistic(self.tspan, rv)
+        random_direction = 1 + 0.1 * np.random.rand(lg1.dimension)
+        random_point = 1 + np.random.rand(lg1.dimension)
+        fd_approx = (
+            0.5
+            * 1e11
+            * (
+                lg1(0.1, random_point + 1e-11 * random_direction)
+                - lg1(0.1, random_point - 1e-11 * random_direction)
+            )
+        )
+        self.assertAllClose(
+            lg1.jacobian(0.1, random_point) @ random_direction, fd_approx, rtol=1e-2
+        )
+
+    def test_fitzhughnagumo(self):
+        """Test the FHN IVP convenience function."""
+        rv = Constant(np.ones(2))
+        lg1 = ivp_examples.fitzhughnagumo(self.tspan, rv)
+        self.assertEqual(issubclass(type(lg1), ivp.IVP), True)
+        lg2 = ivp_examples.fitzhughnagumo(self.tspan, rv, params=(1.0, 1.0, 1.0, 1.0))
+        self.assertEqual(issubclass(type(lg2), ivp.IVP), True)
+
+    def test_fitzhughnagumo_jacobian(self):
+        rv = Constant(np.ones(2))
+        lg1 = ivp_examples.fitzhughnagumo(self.tspan, rv)
+        random_direction = 1 + 0.1 * np.random.rand(lg1.dimension)
+        random_point = 1 + np.random.rand(lg1.dimension)
+        fd_approx = (
+            0.5
+            * 1e11
+            * (
+                lg1(0.1, random_point + 1e-11 * random_direction)
+                - lg1(0.1, random_point - 1e-11 * random_direction)
+            )
+        )
+        self.assertAllClose(
+            lg1.jacobian(0.1, random_point) @ random_direction, fd_approx, rtol=1e-2
+        )
+
+    def test_lotkavolterra(self):
+        """Test the LV ODE convenience function."""
+        rv = Constant(np.ones(2))
+        lg1 = ivp_examples.lotkavolterra(self.tspan, rv)
+        lg2 = ivp_examples.lotkavolterra(self.tspan, rv, params=(1.0, 1.0, 1.0, 1.0))
+        self.assertEqual(issubclass(type(lg2), ivp.IVP), True)
+
+    def test_lotkavolterra_jacobian(self):
+        rv = Constant(np.ones(2))
+        lg1 = ivp_examples.lotkavolterra(self.tspan, rv)
+        random_direction = 1 + 0.1 * np.random.rand(lg1.dimension)
+        random_point = 1 + np.random.rand(lg1.dimension)
+        fd_approx = (
+            0.5
+            * 1e11
+            * (
+                lg1(0.1, random_point + 1e-11 * random_direction)
+                - lg1(0.1, random_point - 1e-11 * random_direction)
+            )
+        )
+        self.assertAllClose(
+            lg1.jacobian(0.1, random_point) @ random_direction, fd_approx, rtol=1e-2
+        )
+
+    def test_seir(self):
+        """Test the SEIR ODE convenience function."""
+        rv = Constant(np.array([1.0, 0.0, 0.0, 0.0]))
+        lg1 = ivp_examples.seir(self.tspan, rv)
+        self.assertEqual(issubclass(type(lg1), ivp.IVP), True)
+        lg2 = ivp_examples.seir(self.tspan, rv, params=(1.0, 1.0, 1.0, 1.0))
+        self.assertEqual(issubclass(type(lg2), ivp.IVP), True)
```

Comment:
why is the seir jacobian not tested?

Reply:
... because there is none, right? If so, my bad :) (why is there none?)

Reply:
There is, now :)